File: Week1/mario1.c
//A program that prints out a half-pyramid of a specified height

#include <cs50.h>
#include <stdio.h>

int main(void)


{
//Prompt user for height
    int height;
    int sp;
    int hsh;
    int i;

    do
    {
        height = get_int("Height: ");
    }
    while (height > 23 || height < 0);

//Row=i

    for (i = 0; i < height; i++)
//Spaces=sp

    {
        for (sp = 0; sp < height - i - 1; sp++)


        {
            printf(" ");
        }


//Hashes=hsh

        for (hsh = 0; hsh < i + 2; hsh++)


        {
            printf("#");
        }



//New line

        printf("\n");
    }

}

File: Week7/mario1.py
# A program that prints out a half-pyramid of a specified height

height = input('Height: ')

while not height or height.isalpha() or int(height) < 0 or int(height) > 23:
    height = input('Height: ')


height = int(height)
for i in range(0, height):
    print((' ' * (height - i - 1)) + ('#' * (i+2)))
